src/transformers/convert_dpr_to_pytorch.py

```diff
@@ -0,0 +1,118 @@
+import argparse
```

Comment:
nitpick but we follow the naming convention `convert_dpr_original_checkpoint_to_pytorch.py`